From 8fb98975d329bcdd08b5e6cc9b2a40feb7a674b5 Mon Sep 17 00:00:00 2001
From: Abhisek Devkota <ciwrl@lineageos.org>
Date: Sun, 11 Feb 2018 23:33:20 -0800
Subject: [PATCH 188/249] manifest: Bring SUW to 15.1

Test cases:
1) Non-gms - pass
2) GMS w/ Pixel2 Google SetupWizard - pass

Change-Id: Ic6d86e1a08716d26299925c8db9dde768e5e7ec0
---
 snippets/lineage.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index cd4497c..e962ee9 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -46,7 +46,7 @@
   <project path="packages/apps/LockClock" name="LineageOS/android_packages_apps_LockClock" />
   <project path="packages/apps/Profiles" name="LineageOS/android_packages_apps_Profiles" />
   <project path="packages/apps/Recorder" name="LineageOS/android_packages_apps_Recorder" />
-  <!--<project path="packages/apps/SetupWizard" name="LineageOS/android_packages_apps_SetupWizard" />-->
+  <project path="packages/apps/SetupWizard" name="LineageOS/android_packages_apps_SetupWizard" />
   <project path="packages/apps/Snap" name="LineageOS/android_packages_apps_Snap" />
   <project path="packages/apps/Trebuchet" name="LineageOS/android_packages_apps_Trebuchet" />
   <project path="packages/apps/Updater" name="LineageOS/android_packages_apps_Updater" />
-- 
2.17.1

